### PR TITLE
Update Chinese Simplified (zh) Upgrades, Community, Tech Stack, Advanced, Learn 2, Tuts 1, and Tuts 3 buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update Chinese Simplified (zh) Upgrades, Community, Tech Stack, Advanced, Learn 2, Tuts 1, and Tuts 3 buckets

### CODE REVIEW

File: src/content/translations/zh/developers/docs/data-and-analytics/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ lang: zh
 
 ## 客户端多样性
 
-[客户端多样性](/developers/docs/nodes-and-clients/client-diversity/)对于以太坊网络的整体健康很重要，因为它提供了对错误和漏洞利用的弹性。 现在有多个客户端多样性仪表板，包括 [clientdiversity.org](https://clientdiversity.org/)、[rated.network](rated.network)、[execution-diversity.info](https://execution-diversity.info/) 和 [Ethernodes](https://ethernodes.org/)。
+[客户端多样性](/developers/docs/nodes-and-clients/client-diversity/)对于以太坊网络的整体健康很重要，因为它提供了对错误和漏洞利用的弹性。 现在有多个客户端多样性仪表板，包括 [clientdiversity.org](https://clientdiversity.org/)、[rated.network](rated.network)、[pool.invis.cloud](pool.invis.cloud)、[slashed.info](slahed.info) 和 [Ethernodes](https://ethernodes.org/)。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/data-and-analytics/index.md:33]:
```suggestion
[客户端多样性](/developers/docs/nodes-and-clients/client-diversity/)对于以太坊网络的整体健康很重要，因为它提供了对错误和漏洞利用的弹性。 现在有多个客户端多样性仪表板，包括 [clientdiversity.org](https://clientdiversity.org/)、[rated.network](rated.network)、[pool.invis.cloud](pool.invis.cloud)、[slashed.info](slashed.info) 和 [Ethernodes](https://ethernodes.org/)。
```

File: src/content/translations/zh/developers/docs/data-availability/index.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ lang: zh
 
 虽然卷叠可以通过链下计算来扩大吞吐量，但它们的容量受到底层区块链的数据吞吐量限制。 如果卷叠使用以太坊作为数据可用性层，就必须增加它的数据存储和处理能力。
 
-[分片](/roadmap/danksharding/)是一种提议的方法，用来增加以太坊执行层的数据吞吐量。 在分片中，网络分割成若干个子链，每个子链都有一组专门的验证者。
+[分片](/upgrades/shard-chains/)是一种提议的方法，用来增加以太坊执行层的数据吞吐量。 在分片中，网络分割成若干个子链，每个子链都有一组专门的验证者。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/data-availability/index.md:152]:
```suggestion
[分片](/roadmap/danksharding/)是一种提议的方法，用来增加以太坊执行层的数据吞吐量。 在分片中，网络分割成若干个子链，每个子链都有一组专门的验证者。
```

File: src/content/translations/zh/developers/docs/scaling/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ sidebarDepth: 3
 
 区块分片是一个横向分割数据库以扩展网络承载能力的过程。 在以太坊中，区块分片将通过创建新链（称为“分片”）来减少网络拥塞和增加每秒交易量。 这还可以减轻每位验证者的负担，因为他们不再需要处理整个网络的所有交易。
 
-了解更多关于[区块分片](/roadmap/danksharding/)的信息。
+了解更多关于[区块分片](/upgrades/sharding/)的信息。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/scaling/index.md:30]:
```suggestion
了解更多关于[区块分片](/roadmap/danksharding/)的信息。
```

File: src/content/translations/zh/developers/docs/scaling/optimistic-rollups/index.md [hunk 4]
```diff
@@ -229,7 +229,7 @@ ii. 使用乐观卷叠的开发者和项目团队可以利用以太坊的基础
 
 这是一个比较乐观的估计，因为乐观卷叠交易不可能包含以太坊上的整个区块。 但是，它可以大致了解乐观卷叠可以为以太坊用户提供多少可扩展性收益（当前实现可提供高达 2,000 交易每秒的速率）。
 
-在以太坊上引入[数据分片](/roadmap/danksharding/)有望提高乐观卷叠的可扩展性。 由于卷叠交易必须与其他非卷叠交易共享区块空间，因此它们的处理能力受到以太坊主链上的数据吞吐量的限制。 分片将增加可供 L2 链发布每个区块的数据的空间，进一步提高卷叠的吞吐量。
+在以太坊上引入[数据分片](/upgrades/shard-chains/)有望提高乐观卷叠的可扩展性。 由于卷叠交易必须与其他非卷叠交易共享区块空间，因此它们的处理能力受到以太坊主链上的数据吞吐量的限制。 分片将增加可供 L2 链发布每个区块的数据的空间，进一步提高卷叠的吞吐量。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/scaling/optimistic-rollups/index.md:232]:
```suggestion
在以太坊上引入[数据分片](/roadmap/danksharding/)有望提高乐观卷叠的可扩展性。 由于卷叠交易必须与其他非卷叠交易共享区块空间，因此它们的处理能力受到以太坊主链上的数据吞吐量的限制。 分片将增加可供 L2 链发布每个区块的数据的空间，进一步提高卷叠的吞吐量。
```

File: src/content/translations/zh/developers/docs/scaling/plasma/index.md [hunk 5]
```diff
@@ -150,7 +150,7 @@ Plasma、侧链、分片技术有一定的相似度，因为它们都以某种
 
 ### Plasma 与分片 {#plasma-vs-sharding}
 
-Plasma 链和[分片链](/roadmap/danksharding/)都会定期向以太坊主网发布加密证明。 但是，两者具有不同的安全属性。
+Plasma 链和[分片链](/upgrades/sharding/)都会定期向以太坊主网发布加密证明。 但是，两者具有不同的安全属性。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/scaling/plasma/index.md:153]:
```suggestion
Plasma 链和[分片链](/roadmap/danksharding/)都会定期向以太坊主网发布加密证明。 但是，两者具有不同的安全属性。
```

File: src/content/translations/zh/developers/docs/scaling/validium/index.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ Volitions 结合了零知识卷叠和 Validium 链，它允许用户在两种扩
 
 ### 1. 链下数据存储 {#off-chain-data-storage}
 
-二层网络扩容项目（例如乐观卷叠和零知识卷叠）通过将部分交易数据发布到一层网络，牺牲了纯链下扩容协议（例如 [Plasma](/developers/docs/scaling/plasma/)）的无限可扩展性来换取安全性。 然而，这意味着卷叠的可扩展性属性受到以太坊主网上的数据带宽的限制（为此，[数据分片](/roadmap/danksharding/)提议是要提高以太坊的数据存储容量）。
+二层网络扩容项目（例如乐观卷叠和零知识卷叠）通过将部分交易数据发布到一层网络，牺牲了纯链下扩容协议（例如 [Plasma](/developers/docs/scaling/plasma/)）的无限可扩展性来换取安全性。 然而，这意味着卷叠的可扩展性属性受到以太坊主网上的数据带宽的限制（为此，[数据分片](/upgrades/sharding/)提议是要提高以太坊的数据存储容量）。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/scaling/validium/index.md:124]:
```suggestion
二层网络扩容项目（例如乐观卷叠和零知识卷叠）通过将部分交易数据发布到一层网络，牺牲了纯链下扩容协议（例如 [Plasma](/developers/docs/scaling/plasma/)）的无限可扩展性来换取安全性。 然而，这意味着卷叠的可扩展性属性受到以太坊主网上的数据带宽的限制（为此，[数据分片](/roadmap/danksharding/)提议是要提高以太坊的数据存储容量）。
```

File: src/content/translations/zh/developers/docs/mev/index.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ lang: zh
 
 ### 矿工可提取价值 {#miner-extractable-value}
 
-最大可提取价值首先应用于[工作量证明](/developers/docs/consensus-mechanisms/pow/)背景下，最初称为“矿工可提取价值”。 这是因为在工作量证明中，矿工掌握了交易的包含、排除和顺序。 然而，自从通过[合并](/roadmap/merge)过渡到权益证明以来，验证者一直负责这些角色，并且挖矿不再是以太坊协议的一部分。 但是，价值提取方法仍然存在，因此现在使用的是术语“最大可提取价值”。
+最大可提取价值首先应用于[工作量证明](/developers/docs/consensus-mechanisms/pow/)背景下，最初称为“矿工可提取价值”。 这是因为在工作量证明中，矿工掌握了交易的包含、排除和顺序。 然而，自从通过[合并](/upgrades/merge)过渡到权益证明以来，验证者一直负责这些角色，并且挖矿不再是以太坊协议的一部分。 但是，价值提取方法仍然存在，因此现在使用的是术语“最大可提取价值”。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/mev/index.md:11]:
```suggestion
最大可提取价值首先应用于[工作量证明](/developers/docs/consensus-mechanisms/pow/)背景下，最初称为“矿工可提取价值”。 这是因为在工作量证明中，矿工掌握了交易的包含、排除和顺序。 然而，自从通过[合并](/roadmap/merge/)过渡到权益证明以来，验证者一直负责这些角色，并且挖矿不再是以太坊协议的一部分。 但是，价值提取方法仍然存在，因此现在使用的是术语“最大可提取价值”。
```

File: src/content/translations/zh/developers/docs/mev/index.md [hunk 8]
```diff
@@ -122,7 +122,7 @@ MEV 并不都是坏事 - 以太坊的 MEV 既有积极的作用，也有消极
 
 在合并后的以太坊中，验证者（已经存入 32 个以太币作为保证金）就添加到信标链的区块的有效性达成共识。 由于 32 个以太币可能超出了许多人的能力范围，[加入质押池](/staking/pools/)也许是一种更可行的选择。 然而，[单独质押人](/staking/solo/)合理分布才是一种理想状态，因为它削弱了验证者的中心化并且提升了以太坊的安全性。
 
-不过，最大可提取价值提取被认为能够加速验证者中心化。 部分原因是，由于验证者[在提出区块时收益要低于](/roadmap/merge/issuance/#how-the-merge-impacts-ETH-supply)当前矿工提出区块的收益，合并后最大可提取价值提取可能会显著[影响验证者的收益](https://github.com/flashbots/eth2-research/blob/main/notebooks/mev-in-eth2/eth2-mev-calc.ipynb)。
+不过，最大可提取价值提取被认为能够加速验证者中心化。 部分原因是，由于验证者[在提出区块时收益要低于](/upgrades/merge/issuance/#how-the-merge-impacts-ETH-supply)当前矿工提出区块的收益，合并后最大可提取价值提取可能会显著[影响验证者的收益](https://github.com/flashbots/eth2-research/blob/main/notebooks/mev-in-eth2/eth2-mev-calc.ipynb)。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/mev/index.md:125]:
```suggestion
不过，最大可提取价值提取被认为能够加速验证者中心化。 部分原因是，由于验证者[在提出区块时收益要低于](/roadmap/merge/issuance/#how-the-merge-impacts-ETH-supply)当前矿工提出区块的收益，合并后最大可提取价值提取可能会显著[影响验证者的收益](https://github.com/flashbots/eth2-research/blob/main/notebooks/mev-in-eth2/eth2-mev-calc.ipynb)。
```

File: src/content/translations/zh/developers/docs/oracles/index.md [hunk 9]
```diff
@@ -360,7 +360,7 @@ contract PriceConsumerV3 {
 
 某些区块链应用程序（如基于区块链的游戏或彩票方案），需要高度的不可预测性和随机性才能有效工作。 然而，区块链的确定性执行消除了任何随机性来源。
 
-通常的方法是使用伪随机密码函数，例如`blockhash`，但这也适用于其他参与者[的](https://ethereum.stackexchange.com/questions/3140/risk-of-using-blockhash-other-miners-preventing-attack#:~:text=So%20while%20the%20miners%20can,to%20one%20of%20the%20players.)操作，即解决工作量证明算法的矿工。 此外，以太坊[过渡到权益证明](/roadmap/merge/)意味着开发者不能再依靠 `blockhash` 获得链上随机性（然而，信标链的 [RANDAO 机制](https://eth2book.info/altair/part2/building_blocks/randomness)提供了另一种随机性来源）。
+通常的方法是使用伪随机密码函数，例如`blockhash`，但这也适用于其他参与者[的](https://ethereum.stackexchange.com/questions/3140/risk-of-using-blockhash-other-miners-preventing-attack#:~:text=So%20while%20the%20miners%20can,to%20one%20of%20the%20players.)操作，即解决工作量证明算法的矿工。 此外，以太坊[过渡到权益证明](/upgrades/merge/)意味着开发者不能再依靠 `blockhash` 获得链上随机性（然而，信标链的 [RANDAO 机制](https://eth2book.info/altair/part2/building_blocks/randomness)提供了另一种随机性来源）。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/oracles/index.md:363]:
```suggestion
通常的方法是使用伪随机密码函数，例如`blockhash`，但这也适用于其他参与者[的](https://ethereum.stackexchange.com/questions/3140/risk-of-using-blockhash-other-miners-preventing-attack#:~:text=So%20while%20the%20miners%20can,to%20one%20of%20the%20players.)操作，即解决工作量证明算法的矿工。 此外，以太坊[过渡到权益证明](/roadmap/merge/)意味着开发者不能再依靠 `blockhash` 获得链上随机性（然而，信标链的 [RANDAO 机制](https://eth2book.info/altair/part2/building_blocks/randomness)提供了另一种随机性来源）。
```

File: src/content/translations/zh/developers/tutorials/how-to-write-and-deploy-an-nft/index.md [hunk 10]
```diff
@@ -239,7 +239,7 @@ Alchemy 非常自豪能够推动非同质化代币领域的一些巨头，包括
 
 为了将它们实际连接到我们的代码，我们将在步骤 13 中在 hardhat.config.js 文件中引用这些变量。
 
-<InfoBanner isWarning>
+<InfoBanner isWarning={true}>
```
Comment (nhsz) [inline src/content/translations/zh/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:242]:
```suggestion
<InfoBanner isWarning>
```

File: src/content/translations/zh/developers/tutorials/kickstart-your-dapp-frontend-development-wth-create-eth-app/index.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ tags:
   - "javascript"
   - "ethers.js"
   - "图表"
-  - "defi"
+  - "deFi"
```
Comment (nhsz) [inline src/content/translations/zh/developers/tutorials/kickstart-your-dapp-frontend-development-wth-create-eth-app/index.md:11]:
```suggestion
  - "defi"
```

File: src/content/translations/zh/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 12]
```diff
@@ -111,7 +111,7 @@ API_URL = "your-api-url"
 PRIVATE_KEY = "your-private-key"
 ```
 
-<InfoBanner isWarning>
+<InfoBanner isWarning={true}>
```
Comment (nhsz) [inline src/content/translations/zh/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:114]:
```suggestion
<InfoBanner isWarning>
```

File: src/content/translations/zh/roadmap/merge/index.md [hunk 13]
```diff
@@ -3,7 +3,7 @@ title: 合并
 description: 了解合并 - 以太坊主网何时采用权益证明机制。
 lang: zh
 template: upgrade
-image: ../../../../../assets/upgrades/merge.png
+image: ../../../assets/upgrades/merge.png
```
Comment (nhsz) [inline src/content/translations/zh/roadmap/merge/index.md:6]:
```suggestion
image: ../../../../../assets/upgrades/merge.png
```

File: src/content/translations/zh/developers/docs/apis/javascript/index.md [hunk 14]
```diff
@@ -10,7 +10,7 @@ lang: zh
 
 如果您想要用 JavaScript 连接到一个以太坊节点， 可以使用原生 JavaScript，不过生态系统中存在一些方便的库，使得这个事情变得更加容易。 通过这些库，开发者可以写下直观易懂甚至单行的代码就能初始化与以太坊的互动（背后使用 JSON RPC 请求）。
 
-请注意，自从[合并](/roadmap/merge/)以来，运行一个节点需要两个连接的以太坊软件 - 一个执行客户端和一个共识客户端。 请确保你的节点同时包含执行客户端和共识客户端。 如果你的节点不在本地计算机上（例如，你的节点在 AWS 实例上运行），请相应地更新教程中的 IP 地址。 有关更多信息，请参阅我们关于[运行节点](/developers/docs/nodes-and-clients/run-a-node/)的页面。
+请注意，自从[合并](/upgrades/merge/)以来，运行一个节点需要两个连接的以太坊软件 - 一个执行客户端和一个共识客户端。 请确保你的节点同时包含执行客户端和共识客户端。 如果你的节点不在本地计算机上（例如，你的节点在 AWS 实例上运行），请相应地更新教程中的 IP 地址。 有关更多信息，请参阅我们关于[运行节点](/developers/docs/nodes-and-clients/run-a-node/)的页面。
```
Comment (nhsz) [inline src/content/translations/zh/developers/docs/apis/javascript/index.md:13]:
```suggestion
请注意，自从[合并](/roadmap/merge/)以来，运行一个节点需要两个连接的以太坊软件 - 一个执行客户端和一个共识客户端。 请确保你的节点同时包含执行客户端和共识客户端。 如果你的节点不在本地计算机上（例如，你的节点在 AWS 实例上运行），请相应地更新教程中的 IP 地址。 有关更多信息，请参阅我们关于[运行节点](/developers/docs/nodes-and-clients/run-a-node/)的页面。
```

File: src/content/translations/zh/glossary/index.md [hunk 15]
```diff
@@ -75,9 +75,9 @@ sidebarDepth: 2
 
 ### 信标链 {#beacon-chain}
 
-信标链是为以太坊引入[权益证明](#pos)和[验证者](#validator)的区块链。 从 2020 年 11 月起，它和工作量证明模式下的以太坊主网并行运行，直到 2022 年 9 月两条链合并，形成现如今的以太坊。
+信标链是为以太坊引入[权益证明](#pos)和[验证者](#validator)的区块链。 从 2020 年 12 月开始，它与采用工作量证明的以太坊主网一起运行，直到 2022 年 9 月这两条链合并，形成了今天的以太坊。
 
-<DocLink to="/roadmap/beacon-chain/">
+<DocLink to="/upgrades/beacon-chain/">
```
Comment (nhsz) [inline src/content/translations/zh/glossary/index.md:80]:
```suggestion
<DocLink to="/roadmap/beacon-chain/">
```

File: src/content/translations/zh/glossary/index.md [hunk 16]
```diff
@@ -238,107 +242,107 @@ Dapp 是去中心化应用程序。 从狭义上来说，去中心化应用程
...
 
 ### 难度 {#difficulty}
 
-一项涉及全网的设置，用来控制在[工作量证明](#pow)网络中产生一个有效随机数所需的计算量。
+[工作量证明](#pow)网络中的全网络设置，用于控制找到有效随机数随机数所需的平均计算量。 难度由产生的区块哈希中成为有效哈希需要的前导零的数量表示。 自权益证明过渡后，这个概念在以太坊中弃用。
 
 ### 难度炸弹 {#difficulty-bomb}
 
-计划的使[工作量证明](#pow)[难度](#difficulty)呈指数级别增长的设置，旨在促进向[权益证明](#pos)的过渡，并减少发生[分叉](#hard-fork)的几率。 难度炸弹在[合并](/roadmap/merge/)时被移除。
+计划的使[工作量证明](#pow)[难度](#difficulty)呈指数级别增长的设置，旨在促进向[权益证明](#pos)的过渡，并减少发生[分叉](#hard-fork)的几率。 难度炸弹在[过渡到权益证明](/upgrades/merge)时弃用。
```
Comment (nhsz) [inline src/content/translations/zh/glossary/index.md:281]:
```suggestion
计划的使[工作量证明](#pow)[难度](#difficulty)呈指数级别增长的设置，旨在促进向[权益证明](#pos)的过渡，并减少发生[分叉](#hard-fork)的几率。 难度炸弹在[过渡到权益证明](/roadmap/merge)时弃用。
```

File: src/content/translations/zh/glossary/index.md [hunk 17]
```diff
@@ -238,107 +242,107 @@ Dapp 是去中心化应用程序。 从狭义上来说，去中心化应用程
...
 
-验证者发送两条相互矛盾的消息。 一个简单的例子是交易发送者发送两笔具有相同随机数的交易。 另一个示例是区块提议者在相同的区块高度（或为相同的时隙）提议两个区块。
+验证者发送两条相互矛盾的信息的情况。 一个简单的例子是交易发送者发送两笔具有相同随机数的交易。 另一个示例是区块提议者在相同的区块高度（或为相同的时隙）提出两个区块。
 
 ### 以太坊 1 {#eth1}
 
 “以太坊 1”是指主网以太坊，即现有的工作量证明区块链。 该术语已弃用，取而代之的是“执行层”。 [详细了解此名称更改](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)。
 
-<DocLink to="/roadmap/">
+<DocLink to="/upgrades/">
```
Comment (nhsz) [inline src/content/translations/zh/glossary/index.md:331]:
```suggestion
<DocLink to="/roadmap/">
```

File: src/content/translations/zh/glossary/index.md [hunk 18]
```diff
@@ -238,107 +242,107 @@ Dapp 是去中心化应用程序。 从狭义上来说，去中心化应用程
...
   有关以太坊升级的更多信息
 </DocLink>
 
 ### 以太坊 2 {#eth2}
 
-“以太坊 2”是指以太坊协议的一系列升级，包括以太坊到权益证明的过渡。 该术语已弃用，取而代之的是“共识层”。 [详细了解此名称更改](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)。
+“以太坊 2”是指以太坊协议的一系列升级，包括以太坊的权益证明过渡。 该术语已弃用，取而代之的是“共识层”。 [详细了解此名称更改](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)。
 
-<DocLink to="/roadmap/">
+<DocLink to="/upgrades/">
```
Comment (nhsz) [inline src/content/translations/zh/glossary/index.md:339]:
```suggestion
<DocLink to="/roadmap/">
```

File: src/content/translations/zh/glossary/index.md [hunk 19]
```diff
@@ -823,7 +823,7 @@ Gigawei 的缩写，[以太币](#ether)的一种计量单位，通常用于[燃
 
 启动了一组扩容和可持续性升级的以太坊开发阶段，以前称为“以太坊 2.0”或“以太坊 2”。
 
-<DocLink to="/roadmap/">
+<DocLink to="/upgrades/">
```
Comment (nhsz) [inline src/content/translations/zh/glossary/index.md:826]:
```suggestion
<DocLink to="/roadmap/">
```

File: src/content/translations/zh/history/index.md [hunk 20]
```diff
@@ -11,31 +11,108 @@ sidebarDepth: 1
...
+分叉是需要对网络进行重大技术升级或更改时的变化——它们通常源自【以太坊改进建议（EIP）】(/eips/) ，并且更改以太坊协议的“规则”。
 
-当传统的中心化软件需要升级时，公司会为终端用户发布新版本。 因为没有中心化所有权，区块链以不同的方式运作。 [以太坊客户端](/developers/docs/nodes-and-clients/) 必须升级他们的软件来执行新的分叉规则。 直链区块创造者（POW 中的矿工，POS 中的验证者）和节点必须创造区块和按照新规则进行验证。 [关于共识机制的更多信息](/developers/docs/consenus-mechanisms/)
+当传统的中心化软件需要升级时，公司会为终端用户发布新版本。 因为没有中心化所有权，区块链以不同的方式运作。 【以太坊客户端】(/developers/docs/nodes-and-clients/) 必须升级他们的软件来执行新的分叉规则。 直链区块创造者（POW 中的矿工，POS 中的验证者）和节点必须创造区块和按照新规则进行验证。 [关于共识机制的更多信息](/developers/docs/consenus-mechanisms/)
 
-这些规则更改可能会在网络中造成暂时的分叉。 新区块的产生，可以来自新规则，也可以来自旧规则。 分叉通常提前商定，以便让客户端能够采用 Unison 的升级，升级后的分叉链成为主链。 然而，在极少数情况下，对分叉的不同意见可能导致网络永久硬分叉——最为著名的是[DAO fork](#dao-fork) 的分叉，产生了 Ethereum Classic（ETC-ETH）。
+这些规则更改可能会在网络中造成暂时的分叉。 新区块的产生，可以来自新规则，也可以来自旧规则。 分叉通常提前商定，以便让客户端能够采用 Unison 的升级，升级后的分叉链成为主链。 然而，在极少数情况下，对分叉的不同意见可能导致网络永久硬分叉——最为著名的是【DAO fork】(#dao-fork) 的分叉，产生了 Ethereum Classic（ETC-ETH）。
+</ExpandableCard>
+
+直接查阅有关一些以往特别重要的升级的信息：[信标链](/upgrades/beacon-chain/)、[合并](/upgrades/merge/)和 [EIP-1559](#london)
```
Comment (nhsz) [inline src/content/translations/zh/history/index.md:21]:
```suggestion
直接查阅有关一些以往特别重要的升级的信息：[信标链](/roadmap/beacon-chain/)、[合并](/roadmap/merge/)和 [EIP-1559](#london)
```

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks, asking for a few fixes

**Review by nhsz — APPROVED** [review]

**Review by corwintines — APPROVED** [review]

**Pull request merged**